oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/components/LocalVocabularySelectField/LocalVocabularySelectField.jsx

```diff
@@ -113,7 +114,7 @@ export const LocalVocabularySelectField = ({
     );
   }
 
-  const serializedOptions = useMemo(
+  let serializedOptions = useMemo(
```

Comment:
Why using memo here, if its later disposed of & reassigned to something else :)

Reply:
@mirekys You are right, will fix it, thanks